src/cmd/hotspots/assert.rs

```diff
@@ -18,16 +18,14 @@ pub struct Cmd {
     gateway: PublicKey,
 
     /// Lattitude of hotspot location to assert. Defaults to the last asserted
-    /// value.
+    /// value. For negative values use '=", for example: "--lat=-xx.xxxxxxx".
```

Comment:
why single quote closed by double quote?

Reply:
good question...I literally didn't do anything but move the commented line up as suggested [here](https://discord.com/channels/404106811252408320/743676367069249536/914982947847819274), so oversight I suppose.